utilities/src/main/java/io/syndesis/qe/TestSuiteParent.java

```diff
@@ -93,4 +119,29 @@ private static void deploySyndesis() {
             CommonSteps.deploySyndesis();
         }
     }
+
+    /**
+     * Update default PostgreSQL connection to use Syndesis DB deployed by test suite
+     */
+    private static void updateDefaultPostgreSQLConnection() {
+        ConnectionsEndpoint connectionsEndpoint = new ConnectionsEndpoint();
+        Connection defaultPostgresDBConnection = connectionsEndpoint.getConnectionByName(DEFAULT_PSQL_CONNECTION_ORIGINAL);
+
+        // backup default connection
+        // (When the test suite was killed before, the backup can exist and original connection was already changed, ignore in that case
+        if (connectionsEndpoint.getConnectionByName(DEFAULT_PSQL_CONNECTION_BACKUP) == null) {
+            connectionsEndpoint.create(defaultPostgresDBConnection.builder().name(DEFAULT_PSQL_CONNECTION_BACKUP).id("1000").build());
+        }
+
+        //config prop for new instance
+        Map<String, String> configuredProperties = new HashMap<>(defaultPostgresDBConnection.getConfiguredProperties());
+        Account account = AccountsDirectory.getInstance().getAccount(Account.Name.SYNDESIS_DB).get();
+        configuredProperties.put("schema", account.getProperty("schema"));
+        configuredProperties.put("password", account.getProperty("password"));
+        configuredProperties.put("user", account.getProperty("user"));
+        configuredProperties.put("url", account.getProperty("url"));
+
+        Connection build = defaultPostgresDBConnection.builder().configuredProperties(configuredProperties).build();
+        connectionsEndpoint.update(defaultPostgresDBConnection.getId().get(), build);
+    }
 }
```

Comment:
make this part of `SyndesisDB#deploy()` and `SyndesisDB#undeploy`?

Reply:
good point, i will try